### ISSUE TITLE
Unable to call .getMessage on exceptiosn

### ISSUE THREAD

**Issue by rymndhng** [borkdude/babashka#312]
**version**

```
❯ ~/Downloads/bb --version
babashka v0.0.79-SNAPSHOT
```

**platform**
osx

**problem**

I'm unable to call .getMessage on exceptions. 

A similar bug was reported and fixed in https://github.com/borkdude/babashka/issues/118. 

**repro**

```
❯ ~/Downloads/bb -e '(try (throw (ex-info "foo" {})) (catch Exception e (.getMessage e)))'

clojure.lang.ExceptionInfo: No matching method getMessage found taking 0 args for class clojure.lang.ExceptionInfo [at line 1, column 52]
```

[ Please provide a minimal working reproduction of the problem ]

**expected behavior**

I should expect the code above to return "foo"

**Comment by borkdude** [comment]
@rymndhng Thanks for reporting. Meanwhile, this works:

```
$ bb '(try (throw (ex-info "foo" {})) (catch Exception e (ex-message e)))'
"foo"
```

**Comment by rymndhng** [comment]
Neat, thanks! I hadn't realized this was defined :)

**Comment by borkdude** [comment]
Fixed with #313.
